README.md

```diff
@@ -64,6 +64,36 @@ context.iopipe.measure('custom', 'init', 'db');
 
 ## Config
 
+#### `autoHttp` (object: optional = {})
+
+Automatically create traces and matching metadata for each http/s call made within your function invocation.
+
+#### `autoHttp.enabled` (bool: optional = true)
+
+HTTP/S auto-tracing is enabled by default.
```

Comment:
Asking @adjohn for clarification on this. Enabling by default allows us to ship with 1 version instead of making a new version 2 which would be a breaking change.

Reply:
@coreylight I don't follow why there's a need for two versions vs one if the default is to be disabled vs enabled.  We discussed and decided that we'd like this in kitchen sink, and it should be disabled by default until we've done thorough testing.   Happy to discuss further, as I am not following here.

Reply:
This still appears to be enabled by default? I don't think a version bump is a breaking change when we're ready to enable it by default in the kitchen sink.